docs/source/quicktour.mdx

```diff
@@ -0,0 +1,181 @@
++ from optimum.graphcore import IPUConfig, IPUTrainer, IPUTrainingArguments
+
+  # Download a pretrained model from the Hub
+  model = AutoModelForXxx.from_pretrained("bert-base-uncased")
+
+  # Define the training arguments
+- training_args = TrainingArguments(
++ training_args = IPUTrainingArguments(
+      output_dir="path/to/save/folder/",
++     ipu_config_name="Graphcore/bert-base-ipu", # Any IPUConfig on the Hub or stored locally
+      ...
+  )
+
+  # Define the configuration to compile and put the model on the IPU
++ ipu_config = IPUConfig.from_pretrained(training_args.ipu_config_name)
+
+  # Initialize the trainer
+- trainer = Trainer(
++ trainer = IPUTrainer(
+      model=model,
++     ipu_config=ipu_config
+      args=training_args,
+      train_dataset=train_dataset
+      ...
+  )
+
+  # Use Graphcore IPU for training!
+  trainer.train()
+```
+
+
+### Optimum Habana
+
+To train transformers on Habana's Gaudi processors, 🤗 Optimum provides a `GaudiTrainer` that is very similar to the [🤗 Transformers trainer](https://huggingface.co/docs/transformers/main_classes/trainer). Here is a simple example:
+
+```diff
+- from transformers import Trainer, TrainingArguments
++ from optimum.habana import GaudiTrainer, GaudiTrainingArguments
+
+  # Download a pretrained model from the Hub
+  model = AutoModelForXxx.from_pretrained("bert-base-uncased")
+
+  # Define the training arguments
+- training_args = TrainingArguments(
++ training_args = GaudiTrainingArguments(
+      output_dir="path/to/save/folder/",
++     use_habana=True,
++     use_lazy_mode=True,
++     gaudi_config_name="Habana/bert-base-uncased",
+      ...
+  )
+
+  # Initialize the trainer
+- trainer = Trainer(
++ trainer = GaudiTrainer(
+      model=model,
+      args=training_args,
+      train_dataset=train_dataset,
+      ...
+  )
+
+  # Use Habana Gaudi processor for training!
+  trainer.train()
+```
+
+### ONNX Runtime
+
+To train transformers with ONNX Runtime's acceleration features, 🤗 Optimum provides a `ORTTrainer` that is very similar to the [🤗 Transformers trainer](https://huggingface.co/docs/transformers/main_classes/trainer). Here is a simple example:
+
+```diff
+- from transformers import Trainer
++ from optimum.onnxruntime import ORTTrainer
+
+  # Download a pretrained model from the Hub
+  model = AutoModelForSequenceClassification.from_pretrained("bert-base-uncased")
+
+  # Create a ONNX Runtime Trainer
+- trainer = Trainer(
++ trainer = ORTTrainer(
+      model=model,
+      args=training_args,
+      train_dataset=train_dataset,
++     feature="sequence-classification", # The model type to export to ONNX
+      ...
+  )
+
+  # Use ONNX Runtime for training!
+  trainer.train()
+```
+
+
+## Accelerated inference
+
+### ONNX Runtime
+
+To accelerate inference with ONNX Runtime, 🤗 Optimum uses _configuration objects_ to define parameters for optimization. These objects are then used to instantiate dedicated _optimizers_, _quantizers_, and _pruners_.
```

Comment:
Are you thinking about INC for "pruners"?

Reply:
Ah yes, this is an artifact from migrating the original quickstart. Fixed in [7d97309](https://github.com/huggingface/optimum/pull/395/commits/7d97309cf66093866e89178c282ebc022baaab46)